net-p2p/go-ethereum/go-ethereum-1.14.8.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module
+
+LONG_VERSION="${PV}+build30152+noble"
+DESCRIPTION="Official golang implementation of the Ethereum protocol"
+HOMEPAGE="https://github.com/ethereum/go-ethereum"
+SRC_URI="https://ppa.launchpadcontent.net/ethereum/ethereum/ubuntu/pool/main/e/ethereum-unstable/ethereum-unstable_${LONG_VERSION}.tar.xz -> ${P}.tar.xz"
```

Comment:
I can't download the package from here.

Reply:
I guess they stabilized 1.14.8 while I wasn't looking. Sorry about that!

Reply:
No need to be sorry, if we were faster handling this we would've gotten it mirrored with our mirror servers. Just unfortunate timing. Will check your updated PR in few hours!